crates/primitives/src/constants.rs

```diff
@@ -1,34 +1,54 @@
-use crate::Address;
+use alloy_primitives::{address, Address};
 
 /// EIP-170: Contract code size limit
-/// By default limit is 0x6000 (~25kb)
+///
+/// By default the limit is `0x6000` (~25kb)
 pub const MAX_CODE_SIZE: usize = 0x6000;
 
-/// Number of block hashes that EVM can access in the past
+/// Number of block hashes that EVM can access in the past (pre-Prague).
 pub const BLOCK_HASH_HISTORY: usize = 256;
 
+/// EIP-2935: Serve historical block hashes from state
+///
+/// Number of block hashes the EVM can access in the past (Prague).
+pub const HISTORY_SERVE_WINDOW: usize = 8192;
+
+/// EIP-2935: Serve historical block hashes from state
+///
+/// The address where historical blockhashes are available.
+pub const HISTORY_STORAGE_ADDRESS: Address = address!("25a219378dad9b3503c8268c9ca836a52427a4fb");
```

Comment:
`HISTORY_` can be ambiguous and the contract needs to be related to `BLOCKHASH` so maybe `BLOCK_HASH_STORAGE_ADDRESS`?

Reply:
sgtm, i just copied over the constant names from the eip, will rename + refer ppl to the original constant name in the docs so its easy to compare with the eip